tidb-cloud/troubleshoot-import-permission-error.md

```diff
@@ -0,0 +1,140 @@
+                }
+            }
+        }
+    ]
+}
+```
+
+In the sample trust entity:
+
+- `380838443567` is the TiDB Cloud Account ID. Make sure that this field in your trust entity matches your TiDB Cloud Account ID.
+- `696e6672612d617069a79c22fa5740944bf8bb32e4a0c4e3fe` is the TiDB Cloud External ID. Make sure that this field in your trusted entity matches your TiDB Cloud External ID.
+
+## Check the Object Ownership
+
+In the AWS Management Console, open the Amazon S3 console, go to **Bucket** > **Permissions**, and find the **Object Ownership** panel. Make sure that the "Object Ownership" configuration is "Bucket owner enforced".
+
+If the configuration is not "Bucket owner enforced", the `AccessDenied` error occurs, because your account does not have enough permissions for all objects in this bucket.
+
+To handle the error, click **edit** on the upper right corner of the panel and change the ownership to "Bucket owner enforced". Note that this might affect your other applications that are using this bucket.
+
+## Check your bucket encryption type
+
+There are multiple ways to encrypt an S3 bucket. When you try to access the objects in a bucket, the role you have created must have the permission to access the encryption key for data decryption. Otherwise, the `AccessDenied` error occurs.
+
+To check the encryption type of your bucket, open the Amazon S3 console in the AWS Management console, choose the name of the target bucket, choose **Properties**, and you will see the **Default encryption** panel that shows the encryption type of the bucket.
+
+On the **Default encryption** panel:
+
+<details>
+<summary>If the server-side encryption is "AWS Key Management Service key (SSE-KMS)" and the AWS KMS key is "AWS managed key (aws/s3)"</summary>
+<br />
+
+In this situation, if the `AccessDenied` error occurs, the reason might be that the key is read-only and cross-account permission grants is not allowed. See the AWS article [Why are cross-account users getting Access Denied errors when they try to access S3 objects encrypted by a custom AWS KMS key](https://aws.amazon.com/premiumsupport/knowledge-center/cross-account-access-denied-error-s3/) for details.
+
+To handle the permission error, click **edit** on the upper right corner of the **Default encryption** panel, and change the AWS KMS key to "Choose from your AWS KMS keys" or "Enter AWS KMS key ARN", or change the server-side encryption method to "AWS S3 Managed Key (SSE-S3).
+</details>
+
+<details>
+<summary>If the server-side encryption is "AWS Key Management Service key (SSE-KMS)" and the AWS KMS key is "Enter AWS KMS key ARN" (customer-managed key)</summary>
+<br />
+
+To handle the `AccessDenied` error in this situation, click the key ARN or manually find the key in KMS. A **Key users** panel is displayed. Click **Add** on the upper right corner of the panel to add the role you have used to import data to TiDB Cloud. Then, retry the data import.
+</details>
+
```

Comment:
```suggestion
Attention please , if the objects was copied from an existing encrypted bucket, the previous key is needed because copy object from an encrypted bucket to another will not change the encrypt method . About more information please refer to [Using default encryption with replication
](https://docs.aws.amazon.com/AmazonS3/latest/userguide/bucket-encryption.html)
```

Reply:
added in Line 146